### PR TITLE
fixed button enable after paper key set

### PR DESCRIPTION
**Pull request by kcw-grunt**

### CODE REVIEW

**Comment by kcw-grunt** [comment]
@losh11 we should update the branch to 2.1.2

**Comment by kcw-grunt** [comment]
@losh11 I wanted to rebase but I didn't want to mess up the branch..

File: breadwallet/src/ApplicationController.swift [hunk 1]
```diff
@@ -268,7 +268,7 @@ class ApplicationController : Subscriber, Trackable {
             self.startDataFetchers()
         })
     }
-    
+    //TODO: What are we losing by not updating these assets?
```
Comment (losh11) [inline breadwallet/src/ApplicationController.swift]: The AssetBundles are a deprecated method (used by BRD) to provide the Support Centre, Buy Bitcoin and EA interface. There's really no point in using this, as we create WebViews which show our custom support centres/etc.

File: breadwallet/src/FlowControllers/StartFlowPresenter.swift [hunk 2]
```diff
@@ -140,13 +118,23 @@ class StartFlowPresenter : Subscriber {
         navigationController?.setClearNavbar()
         navigationController?.pushViewController(pinCreationViewController, animated: true)
     }
-
-    private func handleWalletCreationError() {
-        let alert = UIAlertController(title: S.Alert.error, message: "Could not create wallet", preferredStyle: .alert)
-        alert.addAction(UIAlertAction(title: S.Button.ok, style: .default, handler: nil))
-        navigationController?.present(alert, animated: true, completion: nil)
+    
+    private var pushPinCreationViewForRecoveredWallet: (String) -> Void {
```
Comment (losh11) [inline breadwallet/src/FlowControllers/StartFlowPresenter.swift]: I don't necessarily see the reason for bumping this down, but should be okay.

**Comment by kcw-grunt** [comment]
ok thanks alot.

File: breadwallet/src/FlowControllers/StartFlowPresenter.swift (hunk 2 above)
Comment (kcw-grunt) [reply breadwallet/src/FlowControllers/StartFlowPresenter.swift]: when debugging, reordered these methods and it helps trace the processes the class is getting a bit lengthy.

**Pull request merged**